README.md

```diff
@@ -5,20 +5,22 @@
 [![codecov](https://codecov.io/gh/openkruise/kruise/branch/master/graph/badge.svg)](https://codecov.io/gh/openkruise/kruise)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/2908/badge)](https://bestpractices.coreinfrastructure.org/en/projects/2908)
 
-Kruise is at the core of the OpenKruise project. It is a set of controllers which extends and complements 
+Kruise is the core of the OpenKruise project. It is a set of controllers which extends and complements 
 [Kubernetes core controllers](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/)
-on application workload management.
+on workload management.
 
-Today, Kruise offers three application workload controllers:
+Today, Kruise offers three workload controllers:
 
-* [Advanced StatefulSet](./docs/concepts/astatefulset/README.md): An enhanced version of default [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) with extra functionalities such as `inplace-update`, sharding by namespace.
+* [Advanced StatefulSet](./docs/concepts/astatefulset/README.md): An enhanced version of default [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) with extra functionalities such as `inplace-update`.
 
-* [BroadcastJob](./docs/concepts/broadcastJob/README.md): A job that runs pods to completion across all the nodes in the cluster.
+* [BroadcastJob](./docs/concepts/broadcastJob/README.md): A job that runs Pods to completion across all the nodes in the cluster.
 
-* [SidecarSet](./docs/concepts/sidecarSet/README.md): A controller that injects sidecar container into the pod spec based on selectors.
+* [SidecarSet](./docs/concepts/sidecarSet/README.md): A controller that injects sidecar containers into the Pod spec based on selectors.
 
 Please see [documents](./docs/README.md) for more technical information.
 
+Please find the latest project **roadmap** [here](./docs/roadmap/2019-Q3.md).
```

Comment:
the ideal structure should be 

Roadmap -> ./doc/roadmap/Readme.md which points to the Quarterly roadmaps.

I do not think we will maintain "latest" well here...

Reply:
Done.